### PR TITLE
Fix links and number in product selector

### LINKED ISSUE #9961: Links in product selector are not clickable
@wgx noticed that within the product selectors, it's not possible to click on links, as the whole cards are buttons: 
https://github.com/canonical-web-and-design/ubuntu.com/pull/9940#issuecomment-866954822

That's fine though and it works as intended. I'd suggest to make the link part of the section description, similar to this:

Copy might need some finetuning with @lechsandecki signing off on it.

@ClementChaumel would this be fine for implementing?

### PR DESCRIPTION
**Pull request by ClementChaumel**
## Done

- Took the link out of the cards and put it in the section's subtitle
- Renamed cards to Infra, Apps and Infra + Apps
- replaced periods in number by commas

## QA

- go to https://ubuntu-com-9994.demos.haus/advantage/subscribe?test_backend=true&esm_apps=true
- check that it matches the [design](https://user-images.githubusercontent.com/5436686/124556647-d485b780-de38-11eb-8baf-ed450413c3bc.png)

## Issue / Card

Fixes #9961

## Screenshots

image

### CODE REVIEW

File: templates/advantage/subscribe/form/_feature_coverage.html [hunk 1]
```diff
@@ -3,8 +3,7 @@ <h3 class="p-stepped-list__title">Select your feature coverage</h3>
   <div class="row">
     <div class="col-12 subscribe-section">
       <p>
-        Essential features are already included in your subscription.<br />
-        You can add Infrastructure and Application features to your plan.
+        Essential features are already included in your subscription. <a href="/support">See what is included with Applications coverage.</a>
```
Comment (anthonydillon) [inline templates/advantage/subscribe/form/_feature_coverage.html:6]: I think you should leave the `<br />` in so the link wraps to a new line like the "What Ubuntu version are you running?" section above.

**Review by anthonydillon — APPROVED** [review]
LGTM with a small suggestion and could you update [the copy doc](https://docs.google.com/document/d/1UqL8YJH9ywvOXnrt3syY0U8fiOakfAGWmLS9uerwAWk/edit) to reflect the change, please?

**Comment by ClementChaumel** [comment]
> LGTM with a small suggestion and could you update [the copy doc](https://docs.google.com/document/d/1UqL8YJH9ywvOXnrt3syY0U8fiOakfAGWmLS9uerwAWk/edit) to reflect the change, please?

Done!

File: templates/advantage/subscribe/form/_feature_coverage.html (hunk 1 above)
Comment (wgx) [reply templates/advantage/subscribe/form/_feature_coverage.html]: agree

**Comment by wgx** [comment]
LGTM! 👍🏽

**Pull request merged**